python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_isis.schema.yml

```diff
@@ -82,10 +82,10 @@ keys:
                   - '0'
                   - '7'
                   - '8a'
-                description: Configure authentication key type. Default key_id is 0.
+                description: Configure authentication key type. EOS default `key_type` is 0.
```

Comment:
the default should be '7' here

Reply:
It should be 0 only. Below commands show that eos default is 0.



s1-leaf1(config-if-Vl340)#`isis authentication key testkey `
s1-leaf1(config-if-Vl340)#sh active 
interface Vlan340
   `isis authentication key 7 O61npwCMJwVCgfPTeVATJA==`

s1-leaf1(config-if-Vl340)#no isis authentication key testkey

s1-leaf1(config-if-Vl340)#`isis authentication key 0 testkey`
s1-leaf1(config-if-Vl340)#sh active 
interface Vlan340
  ` isis authentication key 7 O61npwCMJwVCgfPTeVATJA==`

Reply:
It doesn't work with key 7.

s1-leaf1(config-if-Vl340)#isis authentication key 7 testkey
% Unable to decrypt password

Reply:
Yes. I updated the description to `EOS defaut key_type is 7`.

Reply:
well it is not really. On input the default is 0, so when people read this description, they think they can put in an encrypted string without setting the key type. That is not the case. I think we should just remove the comment about EOS default.

Reply:
Removed the line `EOS default key_type is 7`